### ISSUE TITLE
About this SDK

### ISSUE THREAD

**Issue by stafyniaksacha** [prijindal/kuzzle_dart#3]
Hi @prijindal, I was wondering about rewriting this plugin with the new SDK architecture planned for kuzzle.

We can see that there is work in progress about it on the main Javascript SDK here: https://github.com/kuzzleio/sdk-javascript/tree/6-dev

And also a SWIG template is provided in the Go SDK here (but Dartlang is not supported by SWIG yet): https://github.com/kuzzleio/sdk-go/tree/master/internal/wrappers

Here is the work in progress new documentation: 
https://deploy-preview-198--kuzzle-doc-v2.netlify.com/sdk-reference/

What are you thinking about ?

**Comment by prijindal** [comment]
Is the kuzzle API itself is changing or it more like changes to the SDK design?

**Comment by stafyniaksacha** [comment]
Is just a set of changes in SDKs design to reflet the real kuzzle API

**Comment by prijindal** [comment]
Yeah, i think we can definitely do that, it doesn't seem like there are a lot of changes. Just we have to arrange where are functions are at.

**Comment by stafyniaksacha** [comment]
Hi there !

I've requested to add this sdk to the documentation here: https://github.com/kuzzleio/documentation-V2/pull/220

I think we can start publishing this lib to pubspec with alpha tag, and remove it once memoryStorage controller is implemented & tests are good enough. What are you thinking about ?

**Comment by prijindal** [comment]
Yeah we can definitely publish something on pubspec
